app/src/main/res/values/strings.xml

```diff
@@ -270,7 +270,7 @@
     <string name="add_recipient_scan_qr_error">Invalid QR Code.\nYour request cannot be processed at this time.</string>
     <string name="add_recipient_failed_init_camera_message">Failed to initialize Camera</string>
     <string name="add_recipient_camera_permission_denied_message">Camera permission denied</string>
-    <string name="qr_code_scanner_labels_actions_transaction_send">This code contains a funds request.\nWould you like to send funds to %s?</string>
+    <string name="qr_code_scanner_labels_actions_transaction_send">This code contains a funds request.\nWould you like to send funds to \n%s?</string>
```

Comment:
QR phrase is missing here and in localisable keys bellow.

Reply:
@TruszczynskiA What do you mean exactly?